instagram/src/components/CommentSection/Comment.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+
+// class Comment extends React.Component {
+//   state = {
+//     comments: this.props.comments,
+//     newComment: ""
+//   };
+
+//   addNewComment = e => {
+//     e.preventDefault();
+//     const newComment = {
+//       username: "Team Yolo",
+//       text: this.state.newComment
+//     };
+//     this.setState({
+//       comments: [...this.state.comments, newComment],
+//       newComment: ""
+//     });
+//   };
+
+//   changeHandler = e => {
+//     this.setState({
+//       [e.target.name]: e.target.value
+//     });
+//   };
+
+//   render() {
+//     return (
+//       <div>
+//         <div className="comments">
+//           <h4>{this.props.comment.username}</h4>{" "}
+//           <p>{this.props.comment.text}</p>
+//         </div>
+//         {/* <form onSubmit={this.addNewComment}>
+//           <input
+//             type="text"
+//             name="newComment"
+//             value={this.state.newComment}
+//             onChange={this.changeHandler}
+//             placeholder="Add a comment..."
+//           />
+//         </form> */}
+//         ;
+//       </div>
+//     );
+//   }
+// }
+const Comment = props => {
+  console.log(props.comment.username);
+  console.log(props.comment.text);
+  return (
+    <div>
+      {props.comment.username} <p>{props.comment.text}</p>
+    </div>
+  );
```

Comment:
Great job simplifying this, but you could leverage more of the HTML here, like adding 

`@{props.comment.username}: <p>{props.comment.text}</p>{some kind of time stamp}`